### ISSUE TITLE
tokenize incorrectly handles nested single quotes inside double quotes.

### ISSUE THREAD

**Issue by retrogradeorbit** [babashka/process#84]
```bash
$ bash -c "echo 'two words' | wc -w"  
2
```

```clojure
user=> (require '[babashka.process :refer [process check tokenize]])
nil
user=> (-> (process "bash -c \"echo 'two words' | wc -w\"" {:out :string}) deref :out)
"two words\n"
user=> (-> (process ["bash" "-c" "echo 'two words' | wc -w"] {:out :string}) deref :out)
"2\n"
user=> (tokenize "bash -c \"echo 'two words' | wc -w\"")
["bash" "-c" "echo two words" " | wc -w"]
```

`tokenize` should return `["bash" "-c" "echo 'two words' | wc -w"]`

Works the other way around; with doubles inside singles:

```clojure
user=> (tokenize "bash -c 'echo \"two words\" | wc -w'")
["bash" "-c" "echo \"two words\" | wc -w"]
```

**Comment by borkdude** [comment]
```
user=> (tokenize "bash -c \"echo 'two words' | wc -w\"")
["bash" "-c" "echo two words" " | wc -w"]
```
> tokenize should return ["bash" "-c" "echo 'two words' | wc -w"]

Agreed

**Comment by borkdude** [comment]
Fixed in 0.1.7
